lib/base/sysloglogger.cpp

```diff
@@ -61,6 +62,44 @@ void SyslogLogger::StaticInitialize()
 	m_FacilityMap["LOG_UUCP"] = LOG_UUCP;
 }
 
+bool SyslogHelper::ValidateFacility(const String& facility)
+{
+	if (m_FacilityMap.find(facility) == m_FacilityMap.end()) {
+		try {
+			Convert::ToLong(facility);
+		} catch (const std::exception&) {
+			return false;
+		}
+	}
+	return true;
+}
+
+int SyslogHelper::SeverityToNumber(LogSeverity severity)
+{
+	switch (severity) {
+	case LogDebug:
+		return LOG_DEBUG;
+        case LogNotice:
+		return LOG_NOTICE;
+        case LogWarning:
+		return LOG_WARNING;
+	case LogCritical:
+		return LOG_CRIT;
+	case LogInformation:
+        default:
+		return LOG_INFO;
+	}
```

Comment:
There shouldn't be mixed indentation here.

Reply:
Yes, sorry for the noise. Should adapt my editor settings to your style guide - this happens when IDEs try being smart.

Have you considered putting a `.clang-format` in place, for style enforcement? I find such auto-formatting very helpful, esp. when hopping through various projects with different styles.

Reply:
> Have you considered putting a `.clang-format` in place, for style enforcement? I find such auto-formatting very helpful, esp. when hopping through various projects with different styles.

Would be quite nice to have, but the coding style used in this repo doesn't really seem to match any of the presets. so one has to put in quite a bit of effort so that there's no huge "reformat code to new style guide" commit that probably introduces merge conflicts in most open PRs.